web/src/pages/CommentListPage.js

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState } from "react";
+
+import {
+    Box,
+    BreadcrumbGroup,
+    Button,
+    Grid,
+    SpaceBetween,
+    TextContent,
+} from "@cloudscape-design/components";
+import { useParams } from "react-router";
+import CommentTableList from "../components/list/CommentTableList";
+import PropTypes from "prop-types";
+import ListDefinition from "../components/list/list-definitions/types/ListDefinition";
+import RelatedTableList from "../components/list/RelatedTableList";
+import Synonyms from "../synonyms";
+
+export default function CommentListPage(props) {
```

Comment:
I know that a lot of our code is still written in javascript but we are trying to move away from it. In the same spirit see if you can migrate these components to TypeScript.

Reply:
Migrated!